dotCMS/src/main/java/com/dotcms/publisher/business/PublisherAPIImpl.java

```diff
@@ -743,10 +749,16 @@ public void deleteElementFromPublishQueueTable(final String identifier, final lo
 			}
 			dc.loadResult();
 		}catch(Exception e){
-
 			Logger.error(PublisherUtil.class,e.getMessage(),e);
 			throw new DotPublisherException("Unable to delete element "+identifier+" :"+e.getMessage(), e);
 		}
+
+		List<PublishQueueElement> queueElementsAfterDelete = Try.of(()->PublisherAPI.getInstance()
```

Comment:
the PublisherAPI is not being retrieved by APILocator, if not it shouldn't?

Reply:
It never does. It's referenced everywhere like this